### PR TITLE
feat: add ajax table

### PR DESCRIPTION
**Pull request by shadowgate15**

### CODE REVIEW

**Comment by shadowgate15** [comment]
Ready for review!

File: src/ajax-form.js [hunk 1]
```diff
@@ -138,13 +138,19 @@ const ajaxForm = async (ev) => {
       const url = new URL(window.location.href);
       let sort;
       let keyword;
+      let startDate;
+      let endDate;
 
       if (body instanceof FormData) {
         sort = body.get('sort');
         keyword = body.get('keyword');
+        startDate = body.get('startDate');
```
Comment (niftylettuce) [inline src/ajax-form.js]: The form field should be `start_date` not `startDate`, same goes for `endDate` to `end_date`.

File: src/ajax-form.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const ajaxForm = async (ev) => {
   // Initialize spinner
   const spinner = new Spinner($);
 
-  // const api = new Frisbee({
+  // Const api = new Frisbee({
```
Comment (niftylettuce) [inline src/ajax-form.js:32]: I hate it when xo does this haha

**Review by niftylettuce — CHANGES_REQUESTED** [review]
Just needs `start_date` and `end_date` as opposed to `startDate` and `endDate`.  I'm not sure if `.get` will automatically pull `end_date` as `endDate` camelCase.

**Comment by shadowgate15** [comment]
what do you mean. those forms are passed in with the name as `startDate` and `endDate`.

**Pull request merged**